File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

int n;

int main(void)
{
    do
    {
        n = get_int("Height:");
    }
    while (n < 1 || n > 8);
    
 for (int i=0;i<n;i++)
    {
       for (int s=n-1; s>i; s--)
            {
                printf(" ");
            } 
       for (int h = -1; h < i; h++)
        {
            printf("#");
        } 
        printf("\n");
    }
}

File: pset6/sentimental-mario-less/mario.py
#mario-less
from cs50 import get_int

# using a loop to validate input
while True:
    n = get_int("Height: ")
    if (n > 0 and n < 9):
        break

s = 1
while s <= n:
    # print the leading spaces
    for i in range(s, n):
        print(" ", end="")
    # print the pound signs
    print("#" * s)
    s += 1